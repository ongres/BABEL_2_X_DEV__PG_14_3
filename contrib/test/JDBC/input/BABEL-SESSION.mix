-- tsql
CREATE LOGIN r1 WITH PASSWORD = 'abc';
GO

CREATE LOGIN johndoe WITH PASSWORD = 'abc';
GO

CREATE DATABASE db1;
GO

USE db1;
GO

CREATE USER r1;
GO

CREATE TABLE tb1 (a int);
GO

INSERT INTO tb1 (a) VALUES (1);
GO

GRANT SELECT ON tb1 TO r1;
GO

CREATE USER janedoe FOR LOGIN johndoe;
GO

CREATE SCHEMA janedoe_schema;
GO

CREATE TABLE janedoe_schema.t1 (a int);
GO

INSERT INTO janedoe_schema.t1 (a) VALUES (10);
GO

GRANT SELECT ON janedoe_schema.t1 TO janedoe;
GO

USE master;
GO

-- tsql user=r1 password=abc
USE db1;
GO

SELECT db_name();
GO

SELECT user_name();
GO

SELECT schema_name();
GO

SELECT * FROM tb1;
GO

USE master;
GO

-- tsql user=johndoe password=abc
USE db1;
GO

SELECT schema_name();
GO

ALTER USER janedoe WITH DEFAULT_SCHEMA = janedoe_schema;
GO

SELECT schema_name();
GO

SELECT * FROM t1;
GO

USE master;
GO

-- tsql
DROP DATABASE db1;
GO

DROP LOGIN r1;
GO

DROP LOGIN johndoe;
GO
