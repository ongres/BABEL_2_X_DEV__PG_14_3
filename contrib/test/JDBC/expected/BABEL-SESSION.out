-- tsql
CREATE LOGIN r1 WITH PASSWORD = 'abc';
GO

CREATE LOGIN johndoe WITH PASSWORD = 'abc';
GO

CREATE DATABASE db1;
GO

USE db1;
GO

CREATE USER r1;
GO

CREATE TABLE tb1 (a int);
GO

INSERT INTO tb1 (a) VALUES (1);
GO
~~ROW COUNT: 1~~


GRANT SELECT ON tb1 TO r1;
GO

CREATE USER janedoe FOR LOGIN johndoe;
GO

CREATE SCHEMA janedoe_schema;
GO

CREATE TABLE janedoe_schema.t1 (a int);
GO

INSERT INTO janedoe_schema.t1 (a) VALUES (10);
GO
~~ROW COUNT: 1~~


GRANT SELECT ON janedoe_schema.t1 TO janedoe;
GO

USE master;
GO

-- tsql user=r1 password=abc
USE db1;
GO

SELECT db_name();
GO
~~START~~
nvarchar
db1
~~END~~


SELECT user_name();
GO
~~START~~
nvarchar
r1
~~END~~


SELECT schema_name();
GO
~~START~~
varchar
dbo
~~END~~


SELECT * FROM tb1;
GO
~~START~~
int
1
~~END~~


USE master;
GO

-- tsql user=johndoe password=abc
USE db1;
GO

SELECT schema_name();
GO
~~START~~
varchar
dbo
~~END~~


ALTER USER janedoe WITH DEFAULT_SCHEMA = janedoe_schema;
GO

SELECT schema_name();
GO
~~START~~
varchar
janedoe_schema
~~END~~


SELECT * FROM t1;
GO
~~START~~
int
10
~~END~~


USE master;
GO

-- tsql
DROP DATABASE db1;
GO

DROP LOGIN r1;
GO

DROP LOGIN johndoe;
GO
